lib/challenge.js

```diff
@@ -19,7 +19,16 @@
   // - park
 
 // Answer Starts Here
+var computer {
+  // Description our app managers computer characteristics//
+  manufacture: "Dell"
+  speed: '1.96'
+  memory: '900'
+  sendEmail: ["recipient", "address"]
+  togglePower: ["on", "off"]
+  moveMouse: ["left", "right", "up", "down"]
 
+}
```

Comment:
This is a good object literal, but for one thing: if an object literal has multiple properties, the key-value pairs (e.g., `propertyName : 'a string value'`) must be separated by commas. Otherwise, the interpreter will not know where one ends and the next begins.